setup.cfg

```diff
@@ -31,7 +31,9 @@ console_scripts =
    pbench-list-tools = pbench.cli.agent.commands.tools.list:main
    pbench-list-triggers = pbench.cli.agent.commands.triggers.list:main
    pbench-register-tool-trigger = pbench.cli.agent.commands.triggers.register:main
-   pbench-results-push = pbench.cli.agent.commands.results:results_push
```

Comment:
Notice that `pbench-results-push` was the aberration before.

Reply:
See [my comment](https://github.com/distributed-system-analysis/pbench/pull/2325#issuecomment-845149633) below.